### ISSUE TITLE
ruby 1.9.3-p551 compatibility

### ISSUE THREAD

**Issue by blaslaunchlabs** [hvssle/onfido#23]
using ruby.1.9.3 I got an `response_code:nil`, that causes an error in starting rails app. But with 2.0.0 and above its alright. 

```
/Users/launchlabs10/.rvm/gems/ruby-1.9.3-p551/gems/onfido-0.7.0/lib/onfido/errors/onfido_error.rb:6: syntax error, unexpected tLABEL (SyntaxError)
                   response_code: nil,
```

**Comment by dnlserrano** [comment]
Hi @blaslaunchlabs. It makes sense. Keyword arguments are only available starting with [Ruby 2.0](https://www.ruby-lang.org/en/news/2013/02/24/ruby-2-0-0-p0-is-released/).

Do you want to open a PR specifying the required Ruby version for this gem? It is [pretty straightforward](http://guides.rubygems.org/specification-reference/#required_ruby_version).

We can do it, as well. Let us know. 😉

**Comment by hjheath** [comment]
Ruby 1.9.3 is end of life so I am closing this

https://www.ruby-lang.org/en/news/2015/02/23/support-for-ruby-1-9-3-has-ended/
